### PR TITLE
Http forms

### PR DESCRIPTION
**Pull request by frederico-miranda**
This is a pull request opened for receiving a code review. A live version of this project is available at the following link:

[Live version at Heroku](http://peaceful-oasis-60772.herokuapp.com/)

The reference tutorial course for this practice can be found at the following link:

[The Odin Project - Forms - Project 2](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms#project-2-ruby-on-rails-tutorial)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  root to: 'users#index'
+  resources :users
```
[the remark this answers is not in the document]
Comment (frederico-miranda) [reply config/routes.rb]: That issue has been addressed.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to users_path
```
[the remark this answers is not in the document]
Comment (frederico-miranda) [reply app/controllers/users_controller.rb]: That has been addressed.

File: config/routes.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  root to: 'users#index'
+  resources :users, only: [:index, :new, :edit, :create, :update, :show]
```
Comment (efrapp) [inline config/routes.rb]: `[Optional]` There is another cool way to do it if you are using `:only` with a lot of actions like you have now. You can use `:except` and pass the actions you don't want to include:
```
resources :users, except: [:delete]
```
Check this [doc](https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Resources.html).

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    @user = User.new
+  end
+
+  def edit; end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
```
Comment (efrapp) [inline app/controllers/users_controller.rb]: ✅

**Pull request merged**